core/src/main/java/hudson/cli/CreateJobHelper.java

```diff
@@ -0,0 +1,45 @@
+package hudson.cli;
```

Comment:
Please use another package for new classes. It is also not clear why a new class would be needed at all

Reply:
There's no particularly common class between the two existing classes. I'm happy to put it somewhere else. I just need advice.

I used a class because there were two things that were needed and I didn't have a particularly better way to provide both. I'm not used to returning multiple values in Java outside the form of a class. I'm open to suggestions.

Reply:
@oleg-nenashev: to be clear, since @daniel-beck thinks you missed it, I was asking you for advice. What package would you suggest?

How would you make a static helper class work?

I'll note that the merge conflicts for this were someone doing identical cleanup. Had this been merged, they would have only needed to perform the cleanup once.